Sources/StreamChatUI/ChatChannelList/CellActionView.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+/// View which wraps inside `SwipeActionButton` for leading layout
+public class CellActionView: _View {
+    /// Button wrapped inside this ActionView
+    open var actionButton: UIButton?
```

Comment:
Let's create the button here already to simplify the API

Reply:
Added creation of the actionButton :)